contracts/staking/StakingInfo.sol

```diff
@@ -143,13 +143,13 @@ contract StakingInfo {
     }
 
     function logStaked(
-        address signer,
+        bytes memory signerPubkey,
         uint256 validatorId,
         uint256 activationEpoch,
         uint256 amount,
         uint256 total
     ) public onlyStakeManager {
-        emit Staked(signer, validatorId, activationEpoch, amount, total);
+        emit Staked(validatorId, activationEpoch, amount, total, signerPubkey);
```

Comment:
Having a signer in the event also gives flexibility to UI and data processors like Hermione.

Reply:
Added